functionalTests/src/functionalTest/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+base-urls:
+    idam-user: ${IDAM_API_URL:http://localhost:4501}
```

Comment:
That seems confusingly named

Reply:
I didn't change the name of that property - just converted the file from properties to yaml